shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -293,6 +312,33 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   return YES;
 }
 
+- (UniqueAotDataPtr)loadAotData:(std::string)assetsDir {
+  if (!FlutterEngineRunsAOTCompiledDartCode()) {
+    return nullptr;
+  }
+
+  std::filesystem::path assetsFsDir(assetsDir);
+  std::filesystem::path elfFile("app_elf_snapshot.so");
```

Comment:
Why is this elf data and not the `App.framework`? Isn't the latter what the tool is creating and bundling? Or is this an alternate path that's just for the test fixture? (In which case, please comment that here.)